awesome_owl/static/src/todo/todo_list/todo_list.js

```diff
@@ -0,0 +1,40 @@
+/** @odoo-module **/
+
+import { Component, useState } from "@odoo/owl";
+import { TodoItem } from "../todo_item/todo_item";
+import { useAutoFocus } from "../../utils";
+
+export class TodoList extends Component {
+    static template = "awesome_owl.todo_list";
+    static components = { TodoItem };
+
+    setup() {
+        this.todos = useState([]);
+        this.count = useState({ value: 0 });
```

Comment:
It should be clear that you manipulate reactive values. However, I don't understand why `count` has to be reactive. How does it affect UI rendering ?
```suggestion
        this.state = useState({
            todos: [],
            count: 0
        })
```